testing/ef-tests/src/cases/blockchain_test.rs

```diff
@@ -90,50 +91,47 @@ impl Case for BlockchainTestCase {
                     .database_provider_rw()
                     .unwrap();
 
-                // Insert initial test state into the provider.
-                provider.insert_historical_block(
+                // Insert the genesis block and the initial test state into the provider
+                provider.insert_block(
                     SealedBlock::<reth_primitives::Block>::from_sealed_parts(
                         case.genesis_block_header.clone().into(),
                         BlockBody::default(),
                     )
                     .try_recover()
                     .unwrap(),
+                    StorageLocation::Database,
                 )?;
                 case.pre.write_to_db(provider.tx_ref())?;
 
-                // Initialize receipts static file with genesis
-                {
-                    let static_file_provider = provider.static_file_provider();
-                    let mut receipts_writer =
-                        static_file_provider.latest_writer(StaticFileSegment::Receipts).unwrap();
-                    receipts_writer.increment_block(0).unwrap();
-                    receipts_writer.commit_without_sync_all().unwrap();
-                }
-
```

Comment:
If staticfile does make a difference, it might make sense to initialize the transactions static file too for completeness

Reply:
Seems this PR is still faster -- https://github.com/paradigmxyz/reth/pull/15542#issuecomment-2779768262